src/widgets/mainwindow.h

```diff
@@ -93,6 +95,7 @@ private slots:
     bool             _useSIPrefixes;
     QTimer           _purgeTimer;
     int              _purgeTimerCount;
+    QMessageBox      *_nukeCountdownWindow;
```

Comment:
Do you have  a good reason for adding this as a member var? If not please just instantiate it locally when and if needed.